packages/translation/src/manager.ts

```diff
@@ -33,10 +34,13 @@ export class TranslationManager implements ITranslator {
       connector ?? new TranslatorConnector(translationsUrl, serverSettings);
     this._stringsPrefix = stringsPrefix || '';
     this._englishBundle = new Gettext({ stringsPrefix: this._stringsPrefix });
+    this._currentLocale = DEFAULT_LANGUAGE_CODE;
   }
 
   /**
-   * Get the language code of the current locale.
+   * Get the language code (tag) of the current locale.
+   *
+   * It respects RFC5646: https://datatracker.ietf.org/doc/html/rfc5646
```

Comment:
I would mention BCP 47 as a more general standard. I am not sure if we comply with RFC5646 in particular.

Reply:
For what I understand, this is the same document; see https://www.rfc-editor.org/info/rfc5646. For reference, the link set up in the [spellchecker extension readme](https://github.com/jupyterlab-contrib/spellchecker) for BCP47 is to RFC5646 :wink:

Reply:
Yeah, so the thing is "BCP47" is the long-living name which more folks understand because previously it was defined in RFC 4646 and earlier in RFC 3066 and yet earlier in RFC 1766. My point is, I do not remember all RFCs, but "BCP 47" rings a bell immediately.

Reply:
So I learnt something today, BCP (Best Current Practice) is a sub series of RFC (Request For Comment); list of BCPs https://www.ietf.org/rfc/bcp-index.txt
And the current state of BCP47 is RFC4647 and RFC5646:
> 0047 Language Tags. A. Phillips, M. Davis. (Format: TXT=254322 HTML=bytes) (Also RFC4647, RFC5646)

From [wikipedia](https://en.wikipedia.org/wiki/Best_current_practice)

> Each RFC number refers to a specific version of a document Standard Track, but the BCP number refers to the most recent revision of the document. Thus, citations often reference both the BCP number and the RFC number.

So I would say it is better to point to a RFC5646 as it is a stable reference even if it can become obsolete.